src/components/ArticleShare/index.js

```diff
@@ -0,0 +1,43 @@
+import {
+  FacebookShareButton, FacebookIcon,
+  EmailShareButton, EmailIcon,
+  TwitterShareButton, TwitterIcon,
+} from 'react-share';
+import React, { Component } from "react";
+import './ArticleShare.scss';
+import PropTypes from "prop-types";
+
+class ArticleShare extends Component {
+  renderShare = (ShareButton, ShareIcon, props) => (
+    <>
+      <ShareButton
+        className="share-button"
+        {...props}
+      >
+        <ShareIcon size={32} round />
+      </ShareButton>
+      &nbsp;&nbsp;
+    </>
+  );
+
+  render() {
+    const { url, title } = this.props.article;
+    const ahTag = 'AuthorsHaven';
+    return (
+      <span className=" col l8 s12 row valign-wrapper">
+        {this.renderShare(FacebookShareButton, FacebookIcon, { url, quote: title, hashtage: `#${ahTag}` })}
```

Comment:
@mutaimwiti ,
Good job! Other than `LinkedIn` that has slight problems that the team is aware of all the rest are working just fine. On the arguments provided to the `renderShare` function, maybe you can put them in different lines to enhance the readability. Otherwise good work.